dipy/data/fetcher.py

```diff
@@ -323,18 +323,18 @@ def fetcher():
     "fetch_evac_weights",
     pjoin(dipy_home, 'evac'),
     'https://ndownloader.figshare.com/files/',
-    ['40150867'],
+    ['43037191'],
```

Comment:
Because I did not think an old evac model weights would be necessary. I also changed the architecture code a bit, so if someone tries to use the old model weights, it would do unexpected things

Reply:
ok, I see, thanks! I am fine with this